src/mlpack/methods/preprocess/preprocess_scale_main.cpp

```diff
@@ -0,0 +1,174 @@
+/**
+ * @file preprocess_scale_main.cpp
+ * @author jeffin sam
+ *
+ * A CLI executable to scale a dataset.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/prereqs.hpp>
+#include <mlpack/core/util/mlpack_main.hpp>
+#include <mlpack/core/math/random.hpp>
+#include <mlpack/core/util/cli.hpp>
+#include <mlpack/core/data/scaler_methods/max_abs_scaler.hpp>
+#include <mlpack/core/data/scaler_methods/mean_normalization.hpp>
+#include <mlpack/core/data/scaler_methods/min_max_scaler.hpp>
+#include <mlpack/core/data/scaler_methods/pcawhitening.hpp>
+#include <mlpack/core/data/scaler_methods/zcawhitening.hpp>
+#include <mlpack/core/data/scaler_methods/standard_scaler.hpp>
+#include "mlpack/core/data/scaler_methods/scaling_model.hpp"
+
+using namespace mlpack;
+using namespace mlpack::util;
+using namespace mlpack::data;
+using namespace arma;
+using namespace std;
+
+PROGRAM_INFO("Scale Data",
+    // Short description.
+    "A utility to perform feature scaling.",
+    // Long description.
+    "This utility takes a dataset and performs feature scaling using one of "
+    "the six scaler methods namely: 'max_abs_scaler', 'mean_normalization', "
+    "'min_max_scaler' ,'standard_scaler', 'pcawhitening' and 'zcawhitening'."
```

Comment:
You might consider naming the PCA and ZCA techniques `pca_whitening` and `zca_whitening`, so they match the underscores of the other methods.

Reply:
Made changes